react/src/components/atoms/forms/InputCurrency/index.js

```diff
@@ -101,23 +86,17 @@ const Currency = (props) => (
             }
             const numberValue = numbro.unformat(stringValue);
             if (!Number.isNaN(numberValue)) {
-              const newValue = Number.parseFloat(numberValue + props.step).toFixed(2);
+              let newValue;
+              if (direction === 'up') {
```

Comment:
These two directions have the exact same code. Maybe one should be adding and the other subtracting?

Reply:
true